packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -48,12 +49,19 @@ const dropdownMaxWidth = 320.0;
 class StoryOfYourFlexWidget extends StatefulWidget {
   const StoryOfYourFlexWidget(
     this.properties, {
+    this.highlightChild,
+    this.inspectorController,
     Key key,
   })  : assert(properties != null),
         super(key: key);
 
   final FlexLayoutProperties properties;
 
+  // index of child to be highlighted
+  final int highlightChild;
```

Comment:
add a TODO to switch to indicating the RemoteDiagnosticsNode that is highlighted rather than the index.